common/lib/dependabot/pull_request_creator/message_builder.rb

```diff
@@ -192,14 +192,22 @@ def requirement_commit_message_intro
       end
 
       # rubocop:disable Metrics/PerceivedComplexity
+      # rubocop:disable Metrics/CyclomaticComplexity
+      # rubocop:disable Metrics/AbcSize
       def version_commit_message_intro
+        dependency = dependencies.first
+
         return multidependency_property_intro if dependencies.count > 1 && updating_a_property?
 
         return dependency_set_intro if dependencies.count > 1 && updating_a_dependency_set?
 
+        return transitive_multidependency_intro if dependencies.count > 1 &&
+                                                   dependencies.any?(&:top_level?) &&
```

Comment:
We might be able to avoid disabling the rubocop rules & make this more consistent with the other checks if you extracted these to another method. Maybe `updating_top_level_and_transitive_dependencies?`?

Reply:
Fixed in https://github.com/dependabot/dependabot-core/pull/5595/commits/306413f012f903f8dc23721b1e73a044aac45d05

That allowed me to remove the rubocop:disable CyclomaticComplexity line!